### PR TITLE
Queues wizard page minor fixes

### PR DESCRIPTION
**Pull request by tmscarla**
## Description
This PR introduces minor fixes for queues wizard page:

* Changed name structure for queues and compute resources
* Make add/remove button to be repeated in every queue
* Stack EFA/Multithreading checkboxes vertically
* Add missing placeholders for input fields

## How Has This Been Tested?
* Unit tests
* Manually inspected affected pages in local environment, targeting version of PC API 3.2.0 and 3.5.0

## Screenshots

Screenshot 2023-03-22 at 10 23 53

## PR Quality Checklist

- [ ] I added tests to new or existing code
- [x] I removed hardcoded strings and used [`react-i18next`](https://react.i18next.com/) library ([useTranslation hook](https://react.i18next.com/latest/usetranslation-hook) and/or [Trans component](https://react.i18next.com/latest/trans-component)), see an example [here](https://github.com/aws/aws-parallelcluster-ui/commit/a6f1e2aa46b245b5bf7500a04b83195477a5cfa5)
- [ ] I made sure no sensitive info gets logged at any time in the codebase (see [here](https://cheatsheetseries.owasp.org/cheatsheets/Logging_Cheat_Sheet.html)) (e.g. no user info or details, no stacktraces, etc.)
- [ ] I checked that infrastructure/update_infrastructure.sh runs without any error
- [x] I checked that `npm run build` builds without any error
- [x] I checked that clusters are listed correctly
- [ ] I checked that a new cluster can be created (config is produced and dry run passes)
- [ ] I checked that login and logout work as expected

In order to increase the likelihood of your contribution being accepted, please make sure you have read both the [Contributing Guidelines](../CONTRIBUTING.md) and the [Project Guidelines](../PROJECT_GUIDELINES.md)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: frontend/src/old-pages/Configure/Queues/SingleInstanceComputeResource.tsx [hunk 1]
```diff
@@ -240,7 +246,7 @@ export function ComputeResource({index, queueIndex, computeResource}: any) {
 
 export function createComputeResource(queueIndex: number, crIndex: number) {
   return {
-    Name: `queue${queueIndex}-${defaultInstanceType.replace('.', '')}`,
+    Name: `queue-${queueIndex + 1}-${defaultInstanceType.replace('.', '')}`,
```
Comment (mendaomn) [inline frontend/src/old-pages/Configure/Queues/SingleInstanceComputeResource.tsx:249]: issue(non-blocking): same as per the multi instance compute resource

File: frontend/locales/en/strings.json [hunk 2]
```diff
@@ -972,13 +981,15 @@
         "label": "Use placement group"
       },
       "name": {
-        "label": "Queue name"
+        "label": "Queue name",
+        "placeholder": "queue-name"
```
Comment (mendaomn) [inline frontend/locales/en/strings.json:985]: suggestion: probably a better placeholder would include a verb, you can take inspiration from the cluster name "Enter the queue name"

File: frontend/locales/en/strings.json [hunk 3]
```diff
@@ -938,13 +941,19 @@
...
           "label": "Remove resource"
         },
         "addComputeResource": "Add compute resource",
         "staticNodes": "Static nodes",
         "dynamicNodes": "Dynamic nodes",
-        "instanceType": "Instance type",
+        "instanceType": {
+          "label": "Instance type",
+          "placeholder": {
+            "multiple": "Select one or more instance type"
```
Comment (mendaomn) [inline frontend/locales/en/strings.json:954]: typo: I think type should be plural there "Select one or more instance types"

File: frontend/src/old-pages/Configure/Queues/SingleInstanceComputeResource.tsx (hunk 1 above)
Comment (tmscarla) [reply frontend/src/old-pages/Configure/Queues/SingleInstanceComputeResource.tsx:249]: Agree

File: frontend/locales/en/strings.json (hunk 2 above)
Comment (tmscarla) [reply frontend/locales/en/strings.json:985]: Gotcha, thanks

**Review by mendaomn — APPROVED** [review]

**Pull request merged**